enaml/workbench/ui/workbench_window.enaml

```diff
@@ -32,4 +32,4 @@ enamldef WorkbenchWindow(MainWindow):
         Include:
             objects << window_model.menus
     Include:
-        objects << filter(None, [window_model.workspace.content])
+        objects << list(filter(None, [window_model.workspace.content]))
```

Comment:
Commenting here to have you double check this line. I don't feel `list` should be necessary but I don't know the context this is running in, or if this line was computer modified. If it was it left filter's func being None, or is this a manual change & this library handles None being passed to filter?

Reply:
objects needs to be a list (hence the casting for python 3) but an inline if else should work too